### ISSUE TITLE
What are the difference between DINO of IDEA-Research and DINOv from Meta?

### ISSUE THREAD

**Issue by karun2016** [IDEA-Research/DINO#227]
Just a clarification question: What are the main differences between IDEA-Research's DINO (DERT with Improved DeNoising Anchor Boxes for End-to-End Object Detection) vs Meta's DINO self-DIstillation with NO labels  implementation ?

**Comment by FengLi-ust** [comment]
Yes, it is confusing. They are totally two different models just with the same Acronym DINO.
